roles/kubevirt/templates/kubevirt-operator.yaml.j2

```diff
@@ -1,4 +1,11 @@
 ---
+apiVersion: v1
+kind: Namespace
```

Comment:
Remove this namespace creation because it causes deprovision to fail sometimes since the NS is delete before the resources are all cleaned up.